requirements-dev.txt

```diff
@@ -1,6 +1,249 @@
+#
+# This file is autogenerated by pip-compile with python 3.8
+# To update, run:
+#
+#    pip-compile --extra=dev --output-file=requirements-dev.txt --strip-extras setup.cfg
+#
+absl-py==1.1.0
+    # via mediapipe
+asttokens==2.0.5
+    # via stack-data
+attrs==21.4.0
+    # via
+    #   mediapipe
+    #   pytest
+backcall==0.2.0
+    # via ipython
 black==22.3.0
+    # via dot (setup.cfg)
+bump2version==1.0.1
+    # via bumpversion
+bumpversion==0.6.0
+    # via dot (setup.cfg)
+certifi==2022.6.15
+    # via requests
+cfgv==3.3.1
+    # via pre-commit
+chardet==4.0.0
+    # via requests
+click==8.0.2
+    # via
+    #   black
+    #   dot (setup.cfg)
+coverage==6.4.2
+    # via pytest-cov
+cycler==0.11.0
+    # via matplotlib
+decorator==5.1.1
+    # via
+    #   ipdb
+    #   ipython
+distlib==0.3.4
+    # via virtualenv
+dlib==19.19.0
+    # via dot (setup.cfg)
+executing==0.8.3
+    # via stack-data
+face-alignment==1.3.3
+    # via dot (setup.cfg)
+filelock==3.7.1
+    # via virtualenv
 flake8==3.9.2
+    # via dot (setup.cfg)
+flatbuffers==2.0
+    # via onnxruntime
+fonttools==4.34.4
+    # via matplotlib
+identify==2.5.1
+    # via pre-commit
+idna==2.10
+    # via requests
+imageio==2.19.3
+    # via scikit-image
+iniconfig==1.1.1
+    # via pytest
+ipdb==0.13.9
+    # via dot (setup.cfg)
+ipython==8.4.0
+    # via
+    #   dot (setup.cfg)
+    #   ipdb
 isort==5.9.3
+    # via dot (setup.cfg)
+jedi==0.18.1
+    # via ipython
+kiwisolver==1.4.3
+    # via matplotlib
+kornia==0.6.5
+    # via dot (setup.cfg)
+llvmlite==0.38.1
```

Comment:
Change the version of `llvmlite` to `0.38.0` or change the version in [environment-gpu.yaml](https://github.com/sensity-ai/dot/blob/docs/use-mermaid/envs/environment-gpu.yaml) to `0.38.1`

Reply:
ah... i will remove `llvmlite` from the yaml file to avoid conflict.